api/v1/whisker_types.go

```diff
@@ -20,10 +20,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type NotificationStatus string
```

Comment:
```suggestion
type NotificationMode string
```

nit: but Status implies a reported read-only field about cluster-state.

This is a Mode of operation that can be set by the user.

Reply:
Fixed.